docs/databases/users-roles-privileges.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Database users, roles, and privileges
+---
+
+# Database users, roles, and privileges
+
+Metabase needs some [minimum database privileges](#minimum-database-privileges) to successfully connect to your database, as well as some additional database privileges for features like [actions](#actions) and [model caching](#model-caching).
+
+The structure of your database privileges should fit into your [permissions strategy](https://www.metabase.com/learn/permissions/strategy).
+
+## Organize database privileges with roles
```

Comment:
Think we need one more short section, as close to the top of the page as possible, that just gives people a set-and-forget SQL code block they can copy and paste to get moving if they don't want to bother with all this reading and fine-tuning. Something like: create a role that has/lists all the privileges Metabase would need, create a metabase user, grant role to user, done. Like the "Grant all privileges" section, but more explicit about the privileges granted, i.e., this should state every privilege Metabase would need, including `TEMPORARY`, etc. Might even want to use language like "set and forget", but then encourage/recommend the more nuanced approach.

The bonus side effect of frontloading this short-but-explicit version is that the list of grants will let experienced database admins see at a glance exactly what they need, and they can run with it and set up privileges however they want.